mlflow/tensorflow.py

```diff
@@ -694,17 +694,57 @@ def _setup_callbacks(lst):
 def autolog(every_n_iter=100):
     # pylint: disable=E0611
     """
-    Enable automatic logging from TensorFlow to MLflow. If applicable,
-    model checkpoints are logged as artifacts to a 'models' directory, along
-    with any TensorBoard log data.
+    Enables automatic logging from TensorFlow to MLflow.
+    Note that autologging for ``tf.keras`` is handled by :py:func:`mlflow.tensorflow.autolog`,
```

Comment:
Great that we moved this up top! A lot of people were definitely confused by this...